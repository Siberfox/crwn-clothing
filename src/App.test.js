import React from "react";
import { shallow } from "enzyme";
import { App } from "./App";

it("Should render App component", () => {
  expect(shallow(<App />)).toMatchSnapshot();
});
