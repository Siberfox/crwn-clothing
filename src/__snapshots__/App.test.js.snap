// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render App component 1`] = `ShallowWrapper {}`;
